src/vivarium_census_prl_synth_pop/results_processing/addresses.py

```diff
@@ -43,23 +43,36 @@ def get_address_id_maps(
     A dictionary of pd.Series suitable for pd.Series.map, indexed by `address_id`
 
     """
-    if column_name != "address_id":
-        raise ValueError(f"Expected `address_id`, got `{column_name}`")
+    try:
+        output_cols_superset = {
+            "address_id": [column_name, "state_id", "state", "puma", "po_box"],
+            "employer_address_id": [
+                column_name,
+                "employer_state_id",
+                "employer_state",
+                "employer_puma",
+            ],
+        }[column_name]
+    except KeyError:
+        raise ValueError(
+            f"Expected `address_id` or 'employer_address_id', got `{column_name}`"
+        )
+
     maps = dict()
-    output_cols_superset = [column_name, "state_id", "state", "puma", "po_box"]
     formatted_obs_data = format_data_for_mapping(
         index_name=column_name,
         obs_results=obs_data,
         output_columns=output_cols_superset,
     )
     maps.update(get_zipcode_map(column_name, formatted_obs_data, randomness))
-    maps.update(
-        get_household_address_map(column_name, formatted_obs_data, artifact, randomness)
-    )
+    maps.update(get_street_details_map(column_name, formatted_obs_data, artifact, randomness))
     maps.update(get_city_map(column_name, formatted_obs_data, artifact, randomness))
-    maps.update(
-        get_mailing_address_map(column_name, formatted_obs_data, artifact, randomness, maps)
-    )
+    if column_name == "address_id":
+        maps.update(
+            get_mailing_address_map(
+                column_name, formatted_obs_data, artifact, randomness, maps
+            )
+        )
```

Comment:
Maybe just a note saying employers don't have mailing addresses

Reply:
I added the note in address.py when we generate mailing address.